mysql-test/suite/rpl/t/rpl_semi_sync_master_shutdown.test

```diff
@@ -30,8 +30,9 @@ connection master;
 --let $rpl_server_number=1
 source include/rpl_stop_server.inc;
 
---connection slave
---source include/stop_slave.inc
+# After stoping the master, slave receives the disconnect error (2003)
```

Comment:
We should ensure this and wait to see the error before going on, and also the following 2 line comment should be deleted.